crates/bevy_ecs/src/query/fetch.rs

```diff
@@ -1059,12 +1086,14 @@ unsafe impl<'__w, T: Component> WorldQuery for &'__w mut T {
         WriteFetch {
             table_data: None,
             sparse_set: (T::Storage::STORAGE_TYPE == StorageType::SparseSet).then(|| {
-                world
-                    // SAFETY: See &T::init_fetch.
-                    .storages()
-                    .sparse_sets
-                    .get(component_id)
-                    .debug_checked_unwrap()
+                // SAFETY: See &T::init_fetch.
```

Comment:
IMO, safety comments should not have a "see: X" indirection, as they can be very fragile if either call site or invariants change.